src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -26,7 +26,7 @@
 public class CommandInjectionTest {
 
     private CommandInjection commandInjection;
-    private static final String ipAddress = "10.29.28.27";
+    private static final String ipAddress = "127.0.0.1";
     private static final String mockedSuccessPingResponse = "PING SUCCESS";
```

Comment:
can you make both of these constant variable names as LOCALHOST_ADDRESS and MOCKED_SUCCESS_PING_RESPONSE as this is the convention of static final variables